src/test/java/biblioteket/roborally/actors/RobotTest.java

```diff
@@ -104,58 +148,155 @@ void turnRightFromWest() {
     }
 
     @Test
-    void moveForwardWhenFacingNorth() {
-        robot.setDirection(Direction.NORTH);
-        robot.moveForward();
-        assertEquals(robot.getPosition(), new DirVector(0, 1, Direction.NORTH));
+    void moveForwardNorth() {
+        Direction direction = Direction.NORTH;
+        robot.setDirection(direction);
+        DirVector newLocation = robot.getPosition().dirVectorInDirection(direction);
```

Comment:
Especially this, these tests were manual specifically so that they didn't depend on possible errors elsewhere; they singularly tested that the positions were set correctly.